src/olympia/addons/models.py

```diff
@@ -351,6 +351,45 @@ def get_needs_human_review_queue(self, admin_reviewer=False):
                 addonreviewerflags__needs_admin_code_review=True)
         return qs
 
+    def get_mad_queue(self):
```

Comment:
I've not actually attempted it, but this, `get_scanners_queue` and `get_auto_approved_queue` (and possibly `get_content_review_queue`) seem ripe for some refactoring because there's (seemingly) a lot of common code.  

The queryset methods can be built in any order so there could be a method that gets a base queryset for these type of functions with the common queryset modifiers and then the differences added in the individual `get_x_queue` methods.

Reply:
done